de.dlr.sc.virsat.project.test/src/de/dlr/sc/virsat/project/test/AProjectTestCase.java

```diff
@@ -96,8 +98,7 @@ public void tearDown() throws CoreException {
 		}
 
 		if (editingDomain != null) {

```

Comment:
Do we still need the null check? I think we can simplify this by always calling clear and setting the editingDomain to null regardless.

Reply:
Agree! Removed